<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ViewDetails</class>
 <widget class="QMainWindow" name="ViewDetails">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AJ Bank |  View Details</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/background/ViewDetails.jpg);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="MainMenuB">
    <property name="geometry">
     <rect>
      <x>962</x>
      <y>287</y>
      <width>331</width>
      <height>75</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/button/buttons/Main Menu.png);
background-position:center;border-radius:100%;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="CustomerName">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>245</y>
      <width>321</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat ExtraBold</family>
      <pointsize>22</pointsize>
      <weight>10</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/button/buttons/Blue.png);
color: rgb(255, 255, 255);
font: 81 22pt &quot;Montserrat ExtraBold&quot;;</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="CustomerGender">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>305</y>
      <width>321</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat ExtraBold</family>
      <pointsize>22</pointsize>
      <weight>10</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/button/buttons/Blue.png);
color: rgb(255, 255, 255);
font: 81 22pt &quot;Montserrat ExtraBold&quot;;</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="CustomerDOB">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>425</y>
      <width>321</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat ExtraBold</family>
      <pointsize>22</pointsize>
      <weight>10</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/button/buttons/Blue.png);
color: rgb(255, 255, 255);
font: 81 22pt &quot;Montserrat ExtraBold&quot;;</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="CustomerEmailID">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>365</y>
      <width>321</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat ExtraBold</family>
      <pointsize>22</pointsize>
      <weight>10</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/button/buttons/Blue.png);
color: rgb(255, 255, 255);
font: 81 22pt &quot;Montserrat ExtraBold&quot;;</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="CustomerAddress">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>550</y>
      <width>321</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat ExtraBold</family>
      <pointsize>22</pointsize>
      <weight>10</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/button/buttons/Blue.png);
color: rgb(255, 255, 255);
font: 81 22pt &quot;Montserrat ExtraBold&quot;;</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="CustomerPhone">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>490</y>
      <width>321</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat ExtraBold</family>
      <pointsize>22</pointsize>
      <weight>10</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/button/buttons/Blue.png);
color: rgb(255, 255, 255);
font: 81 22pt &quot;Montserrat ExtraBold&quot;;</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BackButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>93</y>
      <width>150</width>
      <height>581</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/button/buttons/Back.png);
border: 0px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="LogoutButton">
    <property name="geometry">
     <rect>
      <x>949</x>
      <y>384</y>
      <width>421</width>
      <height>384</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/button/buttons/Logout.png);
border: 0px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="SuggestButton">
    <property name="geometry">
     <rect>
      <x>949</x>
      <y>0</y>
      <width>421</width>
      <height>384</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/button/buttons/Suggest.png);
border: 0px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="AccountNo">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>60</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/button/buttons/Blue.png);
font: 25 11pt &quot;Montserrat Thin&quot;;
color: rgb(6, 53, 122);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="CustomerBalance">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>610</y>
      <width>321</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat ExtraBold</family>
      <pointsize>22</pointsize>
      <weight>10</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/button/buttons/Blue.png);
color: rgb(255, 255, 255);
font: 81 22pt &quot;Montserrat ExtraBold&quot;;</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
